contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -691,6 +697,11 @@ abstract contract PoSValidatorManager is
             revert InvalidDelegatorStatus(delegator.status);
         }
 
+        // Check that minimum stake duration has passed.
+        if (block.timestamp < delegator.startedAt + $._minimumStakeDuration) {
```

Comment:
Is this check necessary given that we know the delegator is in PendingRemoved state?